lib/pcsclite.js

```diff
@@ -12,7 +11,7 @@ var parse_readers_string = function(readers_str) {
     var pos;
     var readers = [];
     var ini = 0;
-    while ((pos = bt.indexOf(readers_str.slice(ini), '\0')) > 0) {
+    while ((pos = new Buffer(readers_str.slice(ini)).indexOf("\0")) > 0) {
```

Comment:
Thinking about this again, why do you need creating a new `Buffer` if `readers_str.slice()` already returns one?